docs/application/native/guides/multimedia/media-handle.md

```diff
@@ -235,19 +235,19 @@ The following table describes the reference count design of the `media_format_h`
 
 | Function                                 | Reference count number                   | Description                              |
 |------------------------------------------|------------------------------------------|------------------------------------------|
-| `media_format_h fmt1, fmt2, tmp;``media_packet_h pkt1, pkt2;``media_format_create(&fmt1);` | `fmt1`: 1                                | Define the `media_format_h` and `media_packet_h` handles.Create the `fmt1` handle and set the `media_format_video_mime()` or `media_format_audio_mime()` function. |
+| `media_format_h fmt1, fmt2, tmp;`<br> `media_packet_h pkt1, pkt2;`<br> `media_format_create(&fmt1);` | `fmt1`: 1                                | Define the `media_format_h` and `media_packet_h` handles.<br> Create the `fmt1` handle and set the `media_format_video_mime()` or `media_format_audio_mime()` function. |
 | `media_packet_create(&pkt1, fmt1);`      | `fmt1`: 2                                | After the `media_packet_create()` function, you must use the `media_format_unref()` function, because the `media_packet_create()` function increases the `media_format_h` reference count. |
 | `media_format_unref(fmt1);`              | `fmt1`: 1                                | If the `ref_count` is 1, the `fmt1` is currently owned by the `pkt1` only. |
 | `media_packet_copy(pkt1, &pkt2);`        | `fmt1`: 2                                | Copy the `pkt1` metadata to `pkt2`, except the allocated buffer and buffer size. `pkt2` refers to `fmt1`, and `fmt1` `ref_count` is increased. |
 | `media_packet_get_format(pkt1, &tmp);`   | `fmt1`: 3                                | `fmt1` `ref_count` is increased by the `media_packet_get_format()` function. |
 | `media_format_set_video_mime(tmp, ...);` | `fmt1`: 3                                | If you try to modify the `fmt1` data (call the `media_format_set_video_mime()` function) for `fmt1` (=`tmp`), the `ref_count` is bigger than 1, and `fmt1` cannot be modified.To modify the `fmt1` data, call the `media_format_make_writable()` function. |
```

Comment:
line 16: Add a space. (with the media_packet_hhandle.)
line 243: The separated lines are merged as one line.(.To modify the )